src/Microsoft.ApplicationInsights/Extensibility/Implementation/Metrics/DependencyMetricsExtractor.cs

```diff
@@ -88,19 +97,20 @@ public int MaxDependencyTypesToDiscover
                     throw new ArgumentOutOfRangeException(nameof(value), value, "MaxDependencyTypesToDiscover value may not be negative.");
                 }
 
-                this.metricManager?.Flush();
+                this.metricTelemetryClient?.Flush();
                 this.ReinitializeMetrics(value);
             }
         }
 
         /// <summary>
-        /// Initializes the internal metrics trackers based on settings.
+        /// Pre-initialize this extractor.
         /// </summary>
-        /// <param name="metricManager">The <c>MetricManager</c> to be used for creating and sending the metrics by this extractor.</param>
-        public void InitializeExtractor(MetricManagerV1 metricManager)
+        /// <param name="metricTelemetryClient">The <c>TelemetryClient</c> to be used for sending extracted metrics.</param>
+        public void InitializeExtractor(TelemetryClient metricTelemetryClient)
```

Comment:
This looks like a public API contract change from `void init(MetricManager)` to `void init(TelemetryClient)` - is this a breaking change or this class was hidden in 2.6?

Reply:
The class is internal, so no breaking change. Internal classes have public methods if the method represents the contract of the class, but the class is not intended for public consumption.